### ISSUE TITLE
Unable to stream youtube videos

### ISSUE THREAD

**Issue by GauravMohla** [mps-youtube/yewtube#1239]
I am trying to stream a youtube video but it doesn't seem to be working. The program just downloads some chat comments to the download directory.

**Comment by c0dehu1k** [comment]
@GauravMohla Can you please share the URL of youtube video? Is it happening for all the videos?
Can you try upgrading the yt-dlp version?

**Comment by destasetiawan** [comment]
i have same issue like that but my issue is the video take long time to play if the video is live stream 
takes time for more than 5 minutes

**Comment by GauravMohla** [comment]
@c0dehu1k  this happens with all videos. Just updated yt-dlp to latest version and the problem still persists. Is there a verbose/debug mode which can help drill sown this problem. Even after waiting for 15 minutes it does not start streaming. Also, it download chat messages to the yewtube directory whenever I try to stream a live video.

**Comment by c0dehu1k** [comment]
@GauravMohla @destasetiawan Would you like to add some console logs or screenshot to describe the problem further. Also, does yt-dlp works on your system let aside mpyt for the time being.

**Comment by iamtalhaasghar** [comment]
is it happening only with live stream videos? @GauravMohla @destasetiawan

**Comment by destasetiawan** [comment]
@GauravMohla yeah its same with me now live and long video just load too much long

**Comment by bester69** [comment]
hi, live streams query doesnt work.. doesnt filter by live
